SCClassLibrary/Common/Collections/SequenceableCollection.sc

```diff
@@ -64,6 +64,16 @@ SequenceableCollection : Collection {
 		});
 		^obj
 	}
+	*recursiveFormula { arg size, a=0.0;
```

Comment:
What's this method supposed to do? It seems to be not only slightly unrelated to the thread's topic "documenting some undocumented classes and methods"...